FFMediaToolkit/src/Decoding/StreamInfo.cs

```diff
@@ -100,6 +104,27 @@ internal unsafe StreamInfo(AVStream* stream)
         /// </summary>
         public ReadOnlyDictionary<string, string> Metadata { get; }
 
+        /// <summary>
+        /// Gets the stream chapters.
+        /// </summary>
+        public StreamChapter[] Chapters { get; }
```

Comment:
This property should be moved to the `MediaInfo` class.

Reply:
Sorry for the delay, it has been applied.